src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -102,14 +107,17 @@ def map_param_values(
         path: str = self.assign_placeholders(current_request.path, param_values)
 
         headers: Dict[str, Any] = {}
-        for header in current_request.headers or []:
```

Comment:
@seanpreston, added a default value of `[]` for headers and query params to remove the `or []` part from here (as pointed out in a previous PR review)

Reply:
Re-adding `or []` to account for this scenario